addon/-private/state-machine/-base.ts

```diff
@@ -19,33 +14,13 @@ import { scheduleOnce, bind } from '@ember/runloop';
  * @hide
  */
 export default abstract class StateMachine extends EmberObject {
-  /**
-   * @property {A} stepTransitions
-   * @private
-   */
-  stepTransitions: Ember.MutableArray<string> = A();
-
-  /**
-   * @property {string} firstStep
-   * @private
-   */
-  firstStep: string;
+  private stepTransitions: MutableArray<string> = A();
 
-  /**
-   * @property {string} lastStep
-   * @private
-   */
-  lastStep: string;
```

Comment:
Turns out `lastStep` isn't even used anywhere, so we can just remove it altogether.